.luacheckrc

```diff
@@ -0,0 +1,13 @@
+
+unused_args = false
```

Comment:
Not possible at the moment.

This mod incorrectly uses the `function foo:bar(...)` notation (note the colon `:`) when defining functions, which is syntactic sugar for `function foo.bar(self, ...)` (note the period `.`). This makes LuaCheck complain everywhere about an unused `self` argument.

I want to change it to use the correct notation when defining functions in the future (this can be done while maintaining backwards compat), but would like to ask @Jeija, @Ekdohibs, and other big contributors about that first.